### DISCUSSION TITLE
Default Microsoft Translator language support

### DISCUSSION THREAD

**Discussion by lenhutbinh** [immersive-translate/immersive-translate#1428] [Help] [answered]
### 概述 | Summary

Default Bing Translator does not support Vietnamese ?  Last year, when I used the script version I remember it was supported
![Screenshot_20240218-233804](https://github.com/immersive-translate/immersive-translate/assets/125377511/ff427e1a-f7da-4cad-9d8c-e652133a0dfb)

**Comment by keyboard3** [comment]
Next version supports

**Reply by keyboard3** [reply, marked as answer]
1.2.3 Supported
